tests/console/apache_ssl.pm

```diff
@@ -1,20 +1,21 @@
 # SUSE's Apache+SSL tests
 #
-# Copyright 2016-2020 SUSE LLC
+# Copyright 2016-2021 SUSE LLC
 # SPDX-License-Identifier: FSFAP
 #
 # Summary: Enable SSL module on Apache2 server
 #          calls setup_apache2 with mode = SSL (lib/apachetest.pm)
 #
 # Maintainer: Ben Chou <bchou@suse.com>
-# Tags: poo#65375, poo#67309
+# Tags: poo#65375, poo#67309, poo#101782
 
 use base "consoletest";
 use testapi;
 use strict;
 use warnings;
 use apachetest;
 use utils 'clear_console';
+use Utils::Architectures qw(is_aarch64);
```

Comment:
Why is  `Utils::Architectures qw(is_aarch64);` needed here?

Reply:
@lilyeyes you are right, I missed removing this line. Fixed.